lib/ansible/modules/windows/setup.ps1

```diff
@@ -315,7 +315,9 @@ if($gather_subset.Contains('memory')) {
     $ansible_facts += @{
         # Win32_PhysicalMemory is empty on some virtual platforms
         ansible_memtotal_mb = ([math]::ceiling($win32_cs.TotalPhysicalMemory / 1024 / 1024))
-        ansible_swaptotal_mb = ([math]::round($win32_os.TotalSwapSpaceSize / 1024))
```

Comment:
While it is $null we can't just remove it without going through a deprecation cycle. Unfortunately we don't have a good way to do that for return values yet.